lib/bulbs-elements/util/in-view-monitor.js

```diff
@@ -128,7 +138,9 @@ function handleDisplayMutation () {
       });
 
       Object.keys(monitoredItems).forEach((key) => {
-        maybeEmitEvent(monitoredItems[key]);
+        if(monitoredItems[key]) {
```

Comment:
yeah, I was running into errors around `monitoredItems[key]` being undefined and breaking the scroll/resize events